zingolib/src/config.rs

```diff
@@ -726,3 +730,42 @@ impl ActivationHeights {
         }
     }
 }
+
+mod tests {
+
+    #[tokio::test]
+    async fn test_load_clientconfig_serverless() {
+        rustls::crypto::ring::default_provider()
+            .install_default()
+            .expect("Ring to work as a default");
+        tracing_subscriber::fmt().init();
+
+        let valid_uri = crate::config::construct_lightwalletd_uri(Some(
+            crate::config::DEFAULT_LIGHTWALLETD_SERVER.to_string(),
+        ));
```

Comment:
I don't really think this test is correct, I see `valid_uri` with a valid uri, hehehehe, obviously, and IMO this test needs a different value here, IDK like `http::Uri::default()` which for me is a `empty` server uri.